### ISSUE TITLE
Cannot upload photo via Graph API after update from 9.1.1 to 11.1.0

### ISSUE THREAD

**Issue by meor9zcu12** [facebook/facebook-android-sdk#981]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [Github issues](https://github.com/facebook/facebook-android-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Java version

java-1.8.0-openjdk-1.8.0.262

### Android version

Android 7, Android 11

### Android SDK version

11.1.0

### Installation platform & version

Android Gradle Plugin version 4.2.2, Gradle version 6.7.1

### Package

Core & AppEvents

### Goals

Upload photo to Facebook group

### Expected results

{Response:  responseCode: 200, graphObject: {"id":"XXXXXXXXX","post_id":"XXXXXXXXX"}, error: null}

### Actual results

{Response:  responseCode: 400, graphObject: null, error: {HttpStatus: 400, errorCode: 324, subErrorCode: -1, errorType: OAuthException, errorMessage: (#324) Requires upload file}}

### Steps to reproduce

It works fine when using "implementation 'com.facebook.android:facebook-login:9.1.1'"
Keep everything but only update to "implementation 'com.facebook.android:facebook-login:11.1.0'"
Rebuild, install and then test apk.

### Code samples & details

```java
build.gradle
// change from
implementation 'com.facebook.android:facebook-login:9.1.1'
// to
implementation 'com.facebook.android:facebook-login:11.1.0'

Java
GraphResponse response = GraphRequest.newUploadPhotoRequest(
                    AccessToken.getCurrentAccessToken(),
                    String.format("/%s/photos", groupId),
                    uri,
                    null,
                    params,
                    null
            ).executeAndWait();

// print response result

These 3 methods all fail in upload when using 11.1.0
GraphRequest.newUploadPhotoRequest(..., Bitmap bitmap, ...)
GraphRequest.newUploadPhotoRequest(..., Uri uri, ...)
GraphRequest.newUploadPhotoRequest(..., File file, ...)
```

**Comment by linmx0130** [comment]
Hi @meor9zcu12, thanks for raising this issue! This bug is fixed by 260bb5fcb62e2662b0b0b73591d04f407789f4c9 and the fixed patch will be released today in v11.1.1. 

Would you have a test on the new released version?

**Comment by meor9zcu12** [comment]
Yes, v11.1.1 works, thank you
